packages/display/src/DisplayObject.ts

```diff
@@ -685,10 +691,8 @@ export abstract class DisplayObject extends utils.EventEmitter
      */
     destroy(_options?: IDestroyOptions | boolean): void
     {
-        if (this.parent)
-        {
-            this.parent.removeChild(this);
-        }
+        this.parent?.removeChild(this);
```

Comment:
Is there a reason you wouldn't do this instead to keep from duplicating?

```suggestion
        this.removeFromParent();
```